test/e2e/tas/statefulset_test.go

```diff
@@ -46,7 +46,7 @@ var _ = ginkgo.Describe("TopologyAwareScheduling for StatefulSet", func() {
 		gomega.Expect(k8sClient.Create(ctx, ns)).To(gomega.Succeed())
 
 		topology = testing.MakeTopology("datacenter").
-			Levels(topologyLevelBlock, topologyLevelRack, corev1.LabelHostname).
+			Levels(testing.DefaultBlockTopologyLevel, testing.DefaultRackTopologyLevel, corev1.LabelHostname).
 			Obj()
```

Comment:
Let's use the new functions here

Reply:
Changed to `MakeDefaultThreeLevelTopology`